### PR TITLE
use selectors for model example (fixes #942)

### LINKED ISSUE #942: Model test example busted
The scale and position look way off:

[http://localhost:9000/examples/model/](http://localhost:9000/examples/model/)

screenshot 2016-02-05 02 47 45

### PR DESCRIPTION
**Pull request by ngokevin**

### CODE REVIEW

File: examples/model/index.html [hunk 1]
```diff
@@ -14,13 +14,12 @@
         <a-asset-item id="monster" src="models/monster.dae"></a-asset-item>
       </a-assets>
 
-      <a-entity collada-model="url(models/monster.dae)"
-                material="color: green" scale="0.25 0.25 0.25"
+      <a-entity collada-model="#monster" material="color: green" scale="0.25 0.25 0.25"
```
Comment (cvan) [inline examples/model/index.html]: can't do `url` anymore?

Comment (ngokevin) [inline examples/model/index.html]: you can. just more performant and clean to do it this way.

File: examples/model/index.html [hunk 2]
```diff
@@ -14,13 +14,12 @@
...
                 position="-10 -3 1"></a-entity>
-      <a-entity obj-model="obj: url(models/crate/crate.obj); mtl: url(models/crate/crate.mtl)"
-                position="5 -3 5"
 
-      <a-entity position="0 0 15">
+      <a-entity obj-model="obj: #crate-obj; mtl: #crate-mtl" position="5 -3 5">
```
Comment (cvan) [inline examples/model/index.html]: where is this closed?

File: examples/model/index.html (hunk 1 above)
Comment (cvan) [inline examples/model/index.html]: ah, okay, the fix was below, not here

**Pull request merged**